src/routes/single-page-app.js

```diff
@@ -10,6 +10,27 @@ const pageRoutes = require("../data/routes");
 
 const singlePageAppRouter = Router();
 
+singlePageAppRouter.get(
+  "^/locales/:lang([a-zA-Z]{2})/translation.json",
+  (req, res) => {
+    try {
+      const translation = require(`../client/locales/${req.params.lang}/translation.json`);
```

Comment:
Oh, I see. For some reason I thought this was part of the client js and compiled by webpack, but it's not. Should we put the files in src/data/locales instead since it's not actually used by the client?

Also, I wonder if we can use something like etags or last modified to allow some caching by the browser.

Reply:
ok, i'll move the files...we could add etags in another PR